Terminal.Gui/View/ViewDrawing.cs

```diff
@@ -503,5 +503,25 @@ public virtual void OnDrawContentComplete (Rect contentArea)
 			DrawContentComplete?.Invoke (this, new DrawEventArgs (contentArea));
 		}
 
+		/// <summary>
+		/// Draw a frame based on the passed bounds to the screen relative.
+		/// </summary>
+		/// <param name="bounds">The bounds view relative.</param>
+		/// <param name="lineStyle">The line style.</param>
+		/// <param name="attribute">The color to use.</param>
+		public void DrawFrame (Rect bounds, LineStyle lineStyle, Attribute? attribute = null)
+		{
+			var vts = ViewToScreen (bounds);
+			LineCanvas.AddLine (new Point (vts.X, vts.Y), vts.Width,
+				Orientation.Horizontal, lineStyle, attribute);
+			LineCanvas.AddLine (new Point (vts.Right - 1, vts.Y), vts.Height,
+				Orientation.Vertical, lineStyle, attribute);
+			LineCanvas.AddLine (new Point (vts.X, vts.Bottom - 1), vts.Width,
+				Orientation.Horizontal, lineStyle, attribute);
+			LineCanvas.AddLine (new Point (vts.X, vts.Y), vts.Height,
+				Orientation.Vertical, lineStyle, attribute);
+
+			OnRenderLineCanvas ();
```

Comment:
I don't think this call is needed. If you replace it with "SetDisplayNeeded()" the LIneCanvas will be rendered in the natural order of things. RIght?

Reply:
It will have a `mergeWithLineCanvas` parameter as true by default. By default will be merged, otherwise will be rendered immediately.

```cs
			if (mergeWithLineCanvas) {
				LineCanvas.Merge (lc);
			} else {
				OnRenderLineCanvas ();
			}
```